### ISSUE TITLE
The published version of the 0.10.4 contains a dist folder in addon/

### ISSUE THREAD

**Issue by francois2metz** [sinankeskin/ember-dayjs#428]
Hi there !

I was trying to use embroider on our ember applications: http://github.com/1024pix/pix

When using the 0.10.4 version of your addon, I had issues when building the application related to some typescript files:

```
ERROR in ../../../code/orga/node_modules/dayjs/plugin/advancedFormat.d.ts                                                                                                     
Module build failed (from ../../../code/orga/node_modules/thread-loader/dist/cjs.js):                                                                                         
Thread Loader (Worker 1)                                                                                                                                                      
/code/orga/node_modules/dayjs/plugin/advancedFormat.d.ts: Missing semicolon. (3:7)                                                                                            
                                                                                                                                                                              
  1 | import { PluginFunc } from 'dayjs'                                                                                                                                      
  2 |                                                                                                                                                                         
> 3 | declare const plugin: PluginFunc                                                                                                                                        
    |        ^                                                                                                                                                                
  4 | export = plugin                                                                                                                                                         
  5 | 
    at instantiate (/code/orga/node_modules/@babel/parser/lib/index.js:64:32)                                                                                                 
    at constructor (/code/orga/node_modules/@babel/parser/lib/index.js:361:12)                                                                                                
    at Parser.raise (/code/orga/node_modules/@babel/parser/lib/index.js:3251:19)                                                                                              
    at Parser.semicolon (/code/orga/node_modules/@babel/parser/lib/index.js:3622:10)                                                                                          
    at Parser.parseExpressionStatement (/code/orga/node_modules/@babel/parser/lib/index.js:13075:10)                                                                          
    at Parser.parseStatementContent (/code/orga/node_modules/@babel/parser/lib/index.js:12679:19)
    at Parser.parseStatementLike (/code/orga/node_modules/@babel/parser/lib/index.js:12535:17)
    at Parser.parseModuleItem (/code/orga/node_modules/@babel/parser/lib/index.js:12519:17)
    at Parser.parseBlockOrModuleBlockBody (/code/orga/node_modules/@babel/parser/lib/index.js:13106:36)
    at Parser.parseBlockBody (/code/orga/node_modules/@babel/parser/lib/index.js:13099:10)
    at Parser.parseProgram (/code/orga/node_modules/@babel/parser/lib/index.js:12433:10)
    at Parser.parseTopLevel (/code/orga/node_modules/@babel/parser/lib/index.js:12423:25)
    at Parser.parse (/code/orga/node_modules/@babel/parser/lib/index.js:14226:10)
    at parse (/code/orga/node_modules/@babel/parser/lib/index.js:14268:38)
    at parser (/code/orga/node_modules/@babel/core/lib/parser/index.js:41:34)
    at parser.next (<anonymous>)
    at normalizeFile (/code/orga/node_modules/@babel/core/lib/transformation/normalize-file.js:65:38)
    at normalizeFile.next (<anonymous>)
    at run (/code/orga/node_modules/@babel/core/lib/transformation/index.js:21:50)
    at run.next (<anonymous>)
    at transform (/code/orga/node_modules/@babel/core/lib/transform.js:22:41)
    at transform.next (<anonymous>)
    at step (/code/orga/node_modules/gensync/index.js:261:32)
    at /code/orga/node_modules/gensync/index.js:273:13
    at async.call.result.err.err (/code/orga/node_modules/gensync/index.js:223:11)
 @ ./node_modules/ember-dayjs/node_modules/dayjs/plugin/ lazy ^\.\/.*$ namespace object ./advancedFormat.d.ts ./advancedFormat.d
 @ ./node_modules/ember-dayjs/dist/services/dayjs.js 43:35-67
 @ ./assets/pix-orga.js 2891:13-71
```

As typescript is not javascript this make senses. However the `dist/services/dayjs.js` was different than the original `services/dayjs.js` file. It seems to use dynamic import, `await import(\`dayjs/plugin/${plugin}\`);` and webpack try to bundle all files in the plugin dir, so it includes the typescript files.

I guess the `addon/dist` folder was pushed on npm by mistake. Could you publish a new version to fix that?
Btw, I'm looking forward to use the new version with dynamic imports!

**Comment by sinankeskin** [comment]
Whoopsy. I was trying to migrate this addon to v2. I guess there were some leftover folders. I'm handling now. Sorry.

**Comment by sinankeskin** [comment]
I've just released v0.10.6. Should be ok now. Could you please check @francois2metz ? Sorry for inconvenience.

**Comment by francois2metz** [comment]
It worked, thanks for the quick fix. Looking forward for the v2 addon.
